indexes/keys.js

```diff
@@ -0,0 +1,5 @@
+var ViewLevel = require('flumeview-level')
+
+module.exports = function () {
+  return ViewLevel(4, (msg) => [ msg.key ])
+}
```

Comment:
I've pulled all flume indexes into this folder so you can clearly see on what's doing what here